### PR TITLE
Change Install to `pip` in .readthedocs.yaml

### PR DESCRIPTION
**Pull request by jukent**
Our Read the Docs builds are often failing due to excessive memory consumption. According to this [Read-the-Docs Doc](https://docs.readthedocs.io/en/stable/guides/build-using-too-many-resources.html) a recommended work around is to use `pip` instead of `conda` for install whenever possible.

[Example configuration file documentation](https://docs.readthedocs.io/en/stable/config-file/v2.html)

This method seems to only work with a `.txt` file, and fails with `ERROR: Invalid requirement: . . .` when I use the existing `.yaml`. I wonder if there is another way.

### CODE REVIEW

**Comment by jukent** [comment]
The ReadtheDocs build is stalling with 
```
Problem in your project's configuration. Invalid "python.install[0].method": .readthedocs.yaml: Invalid configuration option: python.install.0.method. Make sure the key name is correct.
```
I am unsure what is wrong with my `.yaml` so far.

**Comment by jukent** [comment]
Build failing with "ERROR: Invalid requirement: 'name: pythia' (from line 1 of ci/environment.yml)". Can the environment file not have an environment name if we use `pip`? There has to be a workaround.

**Comment by brian-rose** [comment]
Hmmm it's too bad we're running into limitations building on readthedocs. Ideally we could preview a build of each PR using the same build system that the actual portal site is using. This change would move us farther away from that, since the pip environment won't be identical to the conda-forge environment we're using to build on GitHub. 

I wonder if there's another approach. For the Foundations site, we don't use Readthedocs at all. Rather, we build the PR on GitHub Actions (using the same build environment as our main build) and then display the rendered site on Netlify.

Would the same solution work for the sphinx-based portal site? Would we lose some important functionality that readthedocs provides? I'm not sure.

**Comment by jukent** [comment]
> I wonder if there's another approach. For the Foundations site, we don't use Readthedocs at all. Rather, we build the PR on GitHub Actions (using the same build environment as our main build) and then display the rendered site on Netlify.
> 
> Would the same solution work for the sphinx-based portal site? Would we lose some important functionality that readthedocs provides? I'm not sure.

That sounds better to me, perhaps we discuss this at the next IWG meeting. Unfortunately this is holding up some content PRs as well #147 and #150

**Comment by brian-rose** [comment]
> That sounds better to me, perhaps we discuss this at the next IWG meeting. Unfortunately this is holding up some content PRs as well #147 and #150

Ok, so maybe we should merge this change as a temporary fix but discuss a new strategy for rendering PRs at the next IWG.

**Comment by jukent** [comment]
> Ok, so maybe we should merge this change as a temporary fix but discuss a new strategy for rendering PRs at the next IWG.

That sounds like a good plan to me, maybe we can check in with some other people today before we merge.

**Review by mgrover1 — APPROVED** [review]
This looks good to me! Thanks for making these changes @jukent

File: ci/environment_pip.txt
Comment (dopplershift) [inline ci/environment_pip.txt:7]: Do we actually need precommit for readthedocs?

**Review by kmpaul — APPROVED** [review]
Thanks, @jukent!

File: ci/environment_pip.txt
Comment (jukent) [reply ci/environment_pip.txt:7]: Good question, I just copied all of the dependencies from the `environment.yaml` but maybe if we're having 2 locations for dependencies anyway - they can be different?

**Pull request merged**
